editor/export/project_export.cpp

```diff
@@ -1121,6 +1127,7 @@ void ProjectExportDialog::_export_all(bool p_debug) {
 		platform->clear_messages();
 		Error err = platform->export_project(preset, p_debug, preset->get_export_path(), 0);
 		if (err == ERR_SKIP) {
+			exporting = false;
```

Comment:
This isn't cleared in the `ERR_FAIL` checks above, what would happen in that case?

Reply:
Good point. Ideally, these should never be triggered under normal circumstances. But I guess I should make sure to reset the value nonetheless.

Reply:
Yeah, similar to how we deal with locks on mutexes, could be nice.

In the meantime, extracted checks and made sure to unset the property. Had to add messages to clarity, so added both to this and to the individual export method.